networkstatsquerier/fetcher/cernerlist.go

```diff
@@ -0,0 +1,46 @@
+package fetcher
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// CernerList implements the Endpoints interface for cerner endpoint lists
+type CernerList struct{}
+
+// GetEndpoints takes the list of cerner endpoints and formats it into a ListOfEndpoints
+func (cl CernerList) GetEndpoints(cernerList interface{}) (ListOfEndpoints, error) {
+	var finalList ListOfEndpoints
+	var innerList []EndpointEntry
+	var formatList []map[string]interface{}
+
+	// For reformatting purposes
+	bc, err := json.Marshal(cernerList)
+	if err != nil {
+		return finalList, fmt.Errorf("unable to format cerner list")
+	}
+	err = json.Unmarshal(bc, &formatList)
+	if err != nil {
+		return finalList, fmt.Errorf("unable to format cerner list")
+	}
```

Comment:
i think rather than `Marshal` and `Unmarshal`, you can do `formatList, ok := cernerList.([]map[string]interface{})`, and then check `ok` to make sure it's formatted as expected.

Reply:
i tried that and it didn't work (probably something do with how interfaces are handled?) but i changed how the fetcher unmarshals the original endpoint list, so let me know if the update makes sense!